Userland/Libraries/LibXML/Parser/Parser.cpp

```diff
@@ -545,16 +545,26 @@ ErrorOr<Name, ParseError> Parser::parse_name()
     auto rule = enter_rule();
 
     // Name ::= NameStartChar (NameChar)*
-    auto start = TRY(expect(s_name_start_characters, "a NameStartChar"sv));
+    auto start = m_lexer.tell();
+    StringView remaining = m_lexer.input().substring_view(start);
+    Utf8CodePointIterator code_points = Utf8View { remaining }.begin();
```

Comment:
I don't think the iterator will keep the utf8view alive, maybe:
```suggestion
    auto view = Utf8View { remaining };
    auto code_points = view.begin();
```
?

Reply:
I'm not sure the Utf8View needs to be kept alive since the iterator and view are both just a pointer+length to the original text, but I can add your suggestion anyway.